### PR TITLE
Run3-gem65 Add the demonstration chamber of RPC for the Run3 scenario

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Add the demonstration chamber of RPC for the Run3 scenario

#### PR validation:

Use overlap testing tool to check for overlaps

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Geometry/MuonSimData/data/muonProdCuts/2021/v3/muonProdCuts.xml [hunk 1]
```diff
@@ -20,8 +20,10 @@
    <PartSelector path="//YE1.*"/>
```
Comment (cvuosalo) [inline Geometry/MuonSimData/data/muonProdCuts/2021/v3/muonProdCuts.xml:20]: This version is not in the DB yet, so it is safe to modify.

**Comment by cvuosalo** [comment]
What are the plans for `Geometry/MuonSimData/data/v2/muonSens.xml`?

**Comment by bsunanda** [comment]
It is just a reference - what is in Run3 right now - from that we grow to v3 (include GE21 demonstration) and v4 (also include RPC RR3X and RR4X)

**Comment by cvuosalo** [comment]
@bsunanda With this PR there are two identical files in different directories:
```
Geometry/MuonSimData/data/v2/muonSens.xml
Geometry/MuonSimData/data/muonSens/2021/v2/muonSens.xml
```
Can `Geometry/MuonSimData/data/v2/muonSens.xml` be deleted and all references to it be changed?

**Comment by bsunanda** [comment]
@cvuosalo Yes I shall do that in a separate PR. I did not want to touch that here

**Comment by cvuosalo** [comment]
+1

**Comment by qliphy** [comment]
+1

**Pull request merged**